__tests__/app.test.js

```diff
@@ -121,31 +119,96 @@ describe("Endpoint: GET /api/articles/:article_id", () => {
-//             article_id: expect.any(Number)
-//           });
-//         });
-//       });
-//   });
-// });
+describe("Endpoint: GET /api/articles/:article_id/comments", () => {
+  test("Returns a status code of 200 when query exists", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+  });
+  test("Returns an array of objects with the correct ids and properties", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then((result) => {
+        expect(result.body.comments.length).toBeGreaterThan(1)
+        result.body.comments.forEach((comment) => {
+          expect(comment).toMatchObject({
+            comment_id: expect.any(Number),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+            author: expect.any(String),
+            body: expect.any(String),
+            article_id: expect.any(Number)
+          });
+        });
+      });
+  });
+  test("The comments are sorted in ascending order by their created_at property", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then((res) => {
+        expect(res.body.comments).toBeSortedBy("created_at", {
+          ascending: true,
+        });
+      });
+  });
+  test("Returns 400 when passed an invalid id", () => {
+    return request(app)
+      .get("/api/articles/abc/comments")
+      .expect(400)
+      .then((res) => {
+        expect(res.body.msg).toBe("Bad Request");
+      });
+  });
+});
+
+describe("Endpoint: POST /api/articles/:article_id/comments", () => {
+  test("Returns a status code of 200 when correct post request is made", () => {
+    return request(app)
+      .post("/api/articles/1/comments")
+      .expect(201)
+      .send({
+        username: "butter_bridge",
+            body: "string"
+      })
+  });
+  test("Returns an object with correct properties and values", () => {
+    return request(app)
+    .post("/api/articles/1/comments")
+    .expect(201)
+    .send({
+      username: "butter_bridge",
+      body: "string",
+    })
+    .then((res) => {
+      expect(res.body.article).toMatchObject({
+        comment_id: expect.any(Number),
+        body: expect.any(String),
+        article_id: expect.any(Number),
+        author: expect.any(String),
+        votes: expect.any(Number),
+        created_at: expect.any(String),
+      });
+    })
+  });
+  test("Returns 400 when passed an invalid id", () => {
+    return request(app)
+      .post("/api/articles/abc/comments")
+      .expect(400)
+      .then((res) => {
+        expect(res.body.msg).toBe("Bad Request");
+      });
+  });
+  test("Returns 404 when passed a valid id but it does not exist", () => {
+    return request(app)
+      .get("/api/articles/999/comments")
+      .expect(404)
+      .then((res) => {
+        expect(res.body.msg).toBe("Comments not found");
+      });
+  });
+});
```

Comment:
A few more tests to add: 

201 -> ignore unnecessary properties in object that is sent
404 -> username does not exist
400 -> missing required fields